deploy/example-deployment.yaml

```diff
@@ -39,3 +105,52 @@ spec:
       - name: volume-serving-cert
         secret:
           secretName: cm-adapter-serving-certs
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: api
+  namespace: custom-metrics
+spec:
+  ports:
+  - port: 443
+    targetPort: 6443
+  selector:
+    app: custom-metrics-apiserver
+---
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.custom-metrics.metrics.k8s.io
+spec:
+  insecureSkipTLSVerify: true
```

Comment:
That requires creating a CA cert and using that to sign the serving cert and key.
Totally possible, but will get easier with a helm chart...